moto/ec2/models/instances.py

```diff
@@ -753,6 +754,8 @@ def start_instances(
     ) -> List[Tuple[Instance, InstanceState]]:
         started_instances = []
         for instance in self.get_multi_instances_by_id(instance_ids):
+            if instance.disable_api_stop == "true":
+                raise OperationDisableApiStopNotPermitted(instance.id)
```

Comment:
This should be in `stop_instances` not `start_instances`, right?

Reply:
yes, sorry my bad 🙏🏼  I maybe messed before pushing